apps/06.1Checkers.js

```diff
@@ -53,14 +108,53 @@ function Board() {
   };
 
   // Your code here
+
+
 }
 function Game() {
 
   this.board = new Board();
 
   this.start = function() {
+    this.board.createCheckers(); //this was backwards in the spec  added in class 3/1/2017
     this.board.createGrid();
     // Your code here
+
+    // this.moveChecker = function(start, end) {
+    //   var checker = this.board.selectChecker(start[0], start[1]);
+    //   this.board.grid[start[0]][start[1]] = null;
+    //   this.board.grid[end[0]][end[1]] = checker;
+    //   if (Math.abs(start[0] - end[0]) === 2) {
+    //     var killPosition = [(parseInt(start[0],10) + parseInt(end[0],10)) / 2 , (parseInt(start[1],10) + parseInt(end[1],10)) / 2];
+    //     this.board.killChecker(killPosition);
+    //   }// end of function for moveChecker
+
+
+// notes from Class
+    this.moveChecker = function (start, end){
+
+      var startColumn = start[1];
+
+      var endRow = end[0];
+      var endColumn = end[1];
+
+      this.board.grid[startRow][startColumn] = null;
+
+      this.board.grid[endRow][endColumn] = null;
```

Comment:
This can lead to invalid Checker deletion if a user accidentally tries to drop a checker directly on another checker (instead of "jumping" it)